transports/websocket-websys/src/lib.rs

```diff
@@ -19,27 +19,51 @@
 // SOFTWARE.
 
 //! Libp2p websocket transports built on [web-sys](https://rustwasm.github.io/wasm-bindgen/web-sys/index.html).
+//!
+//! # Node.JS
+//!
+//! This crate will not work out of the box in a Node.JS environment, as there
+//! is no built-in WebSocket implementation, you can fix this by adding the
+//! following line to your project:
+//!
+//! ```js
+//! global.WebSocket = require('isomorphic-ws');
+//! ```
+//!
+//! This will polyfill the `WebSocket` global variable with the `isomorphic-ws`
+//! package.
 
 mod web_context;
 
-use bytes::BytesMut;
```

Comment:
Can we avoid this reformatting please?

Reply:
fixed in [289b0d6](https://github.com/libp2p/rust-libp2p/pull/5025/commits/289b0d6dac0aa29c7c81fab8b849f1731a4d021b)